src/main/kotlin/app/extractors/JavaExtractor.kt

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.extractors
+
+import app.model.DiffContent
+import app.model.Stats
+
+class JavaExtractor : ExtractorInterface {
+    val NAME = "Java"
+
+    val KEYWORDS = listOf("abstract", "continue", "for", "new", "switch",
```

Comment:
Ummm... why are you doing this? We did not make a product decision to parse out keywords. We discussed it as one of several ideas, and while it'a neat idea, we need to have Ryan's buy-in for this.